{"filename": "haskell-enclosed-exceptions-1.0.3-348-x86_64.pkg.tar.zst", "name": "haskell-enclosed-exceptions", "base": "haskell-enclosed-exceptions", "version": "1.0.3-348", "desc": "Catching all exceptions from within an enclosed computation", "csize": "13517", "isize": "40185", "md5sum": "8c2b135e8009ca91747f2b8e770d504d", "sha256sum": "cd8d24be536a698199babf27672cae8ea4447cb16bfd01dfad515bdf8bf1aa44", "pgpsig": "iQIzBAABCgAdFiEEtZcfLFwQqaCMYAMPeGxj8zDXy5IFAmji9jgACgkQeGxj8zDXy5LK4A/9FSKRkeYKT3t/6Ndx0+InIPI3Plww7eGwI2C2Xecgn30cH4qebOvi31X/JlV1SX4D/hv2O9AWOHCnU3FXbNmIhxSKGHu6KcLzBGAKNT93h7fTE/sBxvIoZKqktq7TXPn6QUfIlBEa4bLZYSUtRfU+WLf1K7yMVLlHzYuZsEtD0u0iwWj6pq7dJ2CgFZC/nwBzvKEvDcgUYcM+prmsV2+ukd7n/nwPRV95ScimVyxdLegnSvgujuHGOxG0HYg86ufIV6sVaJ10I1VUoAuDdmgZr98kfuGWR2jFGWLjdC7Ak6we9vlOvvkFG+4VOPhQtyReMubFQfzQvQS0VXknYU5bvsydhgyoj0C/3J3ZbVNWVGMxJ3YQUDmwuvr4ovVmQFvQ9Pl+sfuwz9ih3z/iX72I4XczcR19kJMHUSE0OgSaBbGWaeiPdLQDvBjf+owFvEwzgiF5NGCipaEr+0+q6L5b2cDoY0jAD0dB3a5hl+83LNWpIPs04zCrwmzEQmET1Y2wx4ZSxWRz7KpEXqDwFp7otOn6KmXht3gD6xj9dKFxPbDPr3iWIaiifMobOMZcbkujb0lBWXufWGz5tEuVZlmZ4LHy2tLXq1uRUVGEMX7Xfja1UPatqWOD7mVe9K0E2BpNc/vv4jpQCc+OGSXbjj2Rk6TmYCloNCu+zdD8QfaEJg8=", "url": "https://github.com/jcristovao/enclosed-exceptions", "license": "MIT", "arch": "x86_64", "builddate": "1759704613", "packager": "Felix Yan <felixonmars@archlinux.org>", "depends": ["ghc-libs", "haskell-lifted-base", "haskell-monad-control", "haskell-transformers-base"], "makedepends": ["ghc"]}